queries/period_slippage.sql

```diff
@@ -200,15 +200,20 @@ valued_potential_buffered_trades as (
                                  and date_trunc('minute', block_time) = pusd.minute
 ),
 internal_buffer_trader_solvers as (
-    Select
+    -- See the resulting list at: https://dune.com/queries/908642
+    select
         CONCAT('0x', ENCODE(address, 'hex'))
     from gnosis_protocol_v2."view_solvers"
-    where
-            name = 'DexCowAgg' or
-            name = 'CowDexAg' or
-            name = 'MIP' or
-            name = 'Quasimodo' or
-            name = 'QuasiModo'
+    -- Exclude Single Order Solvers
+    where name not in (
+        '1inch',
+        '0x',
+        'ParaSwap',
+        'Baseline',
```

Comment:
I think the name is "BaselineSolver"

Reply:
While that may be what you call it, that its not its name in Dune. For a list of solver `names`, you can refer to this 

https://github.com/duneanalytics/abstractions/blob/2f6fa2dbebd32e964a969c4eeeec329304dab8d5/ethereum/gnosis_protocol_v2/view_solvers.sql#L26-L67

Reply:
Also, you can look at the PoC query in the Test Plan to see that Baseline is not in the resulting list.

Reply:
Oh right. Sorry about the confusion, my bad. This is how it's called in the driver and I got confused for a bit. Sorry!